packages/providence/src/cli/generate-extend-docs-data.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const fs = require('fs');
+const pathLib = require('path');
+const { performance } = require('perf_hooks');
+
+const { providence } = require('../program/providence.js');
+const { QueryService } = require('../program/services/QueryService.js');
+const { LogService } = require('../program/services/LogService.js');
+
+async function launchProvidenceWithExtendDocs(referencePaths, prefixObj, outputFolder) {
+  const t0 = performance.now();
+
+  const results = await providence(
+    QueryService.getQueryConfigFromAnalyzer('match-paths', { prefix: prefixObj }),
+    {
+      gatherFilesConfig: {
```

Comment:
Good defaults, but this is currently not overridable, do we have plans for providing extensions / exclude rules through CLI or through a providence config object that you can pass? Should it be a blocker for now?

Reply:
Yes, you'r right. Definitely about excludeFolders. Will pick that up